platform/android/CHANGELOG.md

```diff
@@ -18,7 +21,7 @@ Mapbox welcomes participation and contributions from everyone.  If you'd like to
  - Allow map panning after quick zoom is disabled but a phantom gesture is executed [#15093](https://github.com/mapbox/mapbox-gl-native/pull/15093)
```

Comment:
These look like bug fixes but it's hard to tell without clicking on the PR. Could we slightly rephrase all bug fix changelog entries in a format that starts with "Fix", to make it super clear what's being corrected?

p.s. I'm using "Fix" instead of "Fixed" to follow GL-JS documentation convention